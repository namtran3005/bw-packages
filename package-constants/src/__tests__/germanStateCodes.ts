import { germanStateCodes } from '../';

describe('germanStateCodes EN lists', () => {
  it('should match snapshot', () => {
    expect(germanStateCodes).toMatchSnapshot();
  });
});
