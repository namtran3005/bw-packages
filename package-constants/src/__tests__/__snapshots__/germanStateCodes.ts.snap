// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`germanStateCodes EN lists should match snapshot 1`] = `
Array [
  "BW",
  "BY",
  "BE",
  "BB",
  "HB",
  "HH",
  "HE",
  "MV",
  "NI",
  "NW",
  "RP",
  "SL",
  "SN",
  "ST",
  "SH",
  "TH",
]
`;
